src/main/java/com/team4/testingsystem/converters/TestVerificationConverter.java

```diff
@@ -46,6 +51,10 @@ public TestVerificationDTO convertToVerificationDTO(Test test) {
             speakingUrl = null;
         }
 
+        List<CoachGradeDTO> grades = gradeService.getGradesByTest(test.getId()).stream()
+                .map(CoachGradeDTO::new)
+                .collect(Collectors.toList());
```

Comment:
Are we going to send all the grades here?

Reply:
it's only coach grades (grades for speaking and essay)